Docs v2.0 (#3)
nvnieuwk/nf-validation: docs/samplesheets/examples.md

It may be necessary to manipulate this channel to fit your process inputs. For more documentation, check out the [Nextflow operator docs](https://www.nextflow.io/docs/latest/operator.html), however here are some common use cases with `.fromSamplesheet()`.

## Using a sample sheet with no headers

> Sometimes you only have one possible input in the pipeline sample sheet. In this case it doesn't make sense to have a header in the sample sheet. This can be done by removing the `properties` section fro m the samplesheet and changing the type of the element from `object` to whatever type you want:

Reviewer: ```suggestion
Sometimes you only have one possible input in the pipeline sample sheet. In this case it doesn't make sense to have a header in the sample sheet. This can be done by removing the `properties` section from the sample sheet and changing the type of the element from `object` the desired type:
```

This example shows a channel which can have entries for WES or WGS data. WES data includes a BED file denoting the target regions, but WGS data does not. These analysis are different so we want to separate the WES and WGS entries from each other. We can separate the two using `.branch` based on the presence of the BED file:

```groovy
// Channel with four elements - see docs for examples

> params.input = "sample sheet.csv"

Reviewer: ```suggestion
params.input = "samplesheet.csv"
```

Author: Seems like my regex wasn't perfect :grin:

This example contains a channel where multiple samples can be in the same family. Later on in the pipeline we want to merge the analyzed files so one file gets created for each family. The result will be a channel with an extra meta field containing the count of channel entries with the same family name.

```groovy
// channel created by fromSamplesheet() previous to modification:
// [[id:example1, family:family1], example1.txt]
// [[id:example2, family:family1], example2.txt]
// [[id:example3, family:family2], example3.txt]

> params.input = "sample sheet.csv"

Reviewer: ```suggestion
params.input = "samplesheet.csv"
```
